Jenkinsfile_CNP

```diff
@@ -21,8 +21,8 @@ so for testing purposes it makes sense to use as an example.
  */
 def nonPrSecrets = [
   'reform-scan-${env}': [
-    secret('notifications-queue-listen-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_READ'),
-    secret('notifications-queue-send-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_WRITE')
+    secret('notifications-staging-queue-listen-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_READ'),
+    secret('notifications-staging-queue-send-connection-string', 'NOTIFICATION_QUEUE_CONN_STRING_WRITE')
```

Comment:
does functional tests needs both read/write ?

Reply:
they were there. we can keep it now , may need.